3rdparty/p256-m/p256-m/p256-m.h

```diff
@@ -0,0 +1,95 @@
+/*
+ * Interface of curve P-256 (ECDH and ECDSA)
+ *
+ * Author: Manuel Pégourié-Gonnard.
```

Comment:
Should this explicitly state copyright? @mpg

Reply:
Now that you mention it, I think it should.